.github/actions/gatsby-site-showcase-validator/index.js

```diff
@@ -47,34 +48,47 @@ async function run() {
     }
 
     // Pass html into a parser
-    let $ = cheerio.load(siteHtml)
+    const $ = cheerio.load(siteHtml)
 
     // Most Gatsby sites have an id of "___gatsby"
-    let gatsbyContainer = $("#___gatsby")
+    const gatsbyContainer = $("#___gatsby")
 
-    if (gatsbyContainer.length !== 0) {
-      // The site is a gatsby site don't do anything
-      sitesVisited++
-    } else {
-      // The site is not a gatsby site, print out some info
+    // The site is not a gatsby site, print out some info
+    if (!gatsbyContainer.length) {
       console.log(
         `${chalk.yellow(`[Notice]`)}: ${
           site.title
         } (${siteUrl}) is not a Gatsby site`
       )
-      sitesVisited++
       nonGatsbySiteCount++
     }
+
+    // Check if provided repository is public
+    if (sourceUrl) {
+      const status = await fetch(sourceUrl).then(({ status }) => status)
+
+      if (status !== 200) {
+        console.log(
+          `${chalk.yellow(`[Notice]`)}: ${
+            site.title
+          } (${siteUrl}) provided a 'source_url', but it's repository is private (${sourceUrl})`
```

Comment:
it can be private or a wrong url ...

Reply:
should we add an additional check, or modify the message somehow for this case? @lannonbr

Reply:
message is now more generic, thanks for the suggestion